downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc

```diff
@@ -2,7 +2,11 @@
 
 = Overview of tested deployment models
 
-Red Hat tests {PlatformNameShort} {PlatformVers} with a defined set of topologies to give you opinionated deployment options. While it is possible to install the {PlatformNameShort} on different infrastructure topologies and with different environment configurations, Red Hat provides full support for the topologies outlined in this document.
+Red Hat tests {PlatformNameShort} {PlatformVers} with a defined set of topologies to give you opinionated deployment options. Deploy all components of {PlatformNameShort} so that all features and capabilities are available for use without the need to take further action.
+
+Red Hat tests the installation of {PlatformNameShort} {PlatformVers} based on a defined set of infrastructure topologies or reference architectures. Enterprise organizations should use one of the enterprise topologies for production deployments to ensure the highest level of uptime, performance, and continued scalability. Organizations or deployments that are resource constrained can use a "growth" topology.
```

Comment:
Hey I'd recommend avoiding "should" here, maybe "can" or "must" instead?

IBM SG: "Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed. "

Reply:
@g-murray  - Yes, I considered adding that to the edits, but wording, "Whenever possible..." gave me pause. Tried the different options and felt like "should" was tolerable in this instance. Leaving it up to @emurtoug if she wants to make further changes after this PR is merged.